pkg/runtime/reconciler.go

```diff
@@ -1103,6 +1082,55 @@ func (r *resourceReconciler) getEndpointURL(
 	return r.cfg.EndpointURL
 }
 
+// getResyncPeriod returns the period of the recurring reconciler process which ensures the desired
+// state of custom resources is maintained.
+// It attempts to retrieve the duration from the following sources, in this order:
+// 1. A resource-specific reconciliation resync period specified in the reconciliation resync
```

Comment:
why resync configuration map?

can we use annotation on the resource like other existing features e.g.
- https://aws-controllers-k8s.github.io/community/docs/user-docs/multi-region-resource-management/
- https://aws-controllers-k8s.github.io/community/docs/user-docs/deletion-policy/

Reply:
I'm not a super fan of resource-level drift remediation control - but happy to hear what other folks think about it.

Reply:
Ack on the controller flag, my question is related to highest in order to precedence.

Annotation or configmap, both are giving control to the user to configure the resync period. Since we don't have a common configmap or a CRD to define all controller configurations, my preference is to keep the experience consistent and not introduce another place

Reply:
Hi folks, Nick helped me with clarification that resync configuration map != `ConfigMap`, so we can resolve this comment with one suggestion, rename the comment to say `resource resync CLI flag` or something along those lines

also see my comment on No 3 here. We should drop it